### DISCUSSION TITLE
Pixelate Video Filter

### DISCUSSION THREAD

**Discussion by nisaeggu** [silveredgold/beta-protection#21] [Ideas]
Right now videos only can be blocked or blurred, though it would be great to have a pixelate filter too. If possible maybe even adjustable in pixel size.

**Comment by silveredgold** [comment]
This is actually a bit of a browser limitation. Video blurring is not done in real-time by the censoring backend as the performance is far too slow for that. Instead, blurring or hiding videos actually happens entirely in the browser using CSS, and unfortunately the only obfuscation filter that CSS supports is blurring. If browsers ever add support for other methods, we'll probably add them.
